install/01_enabling_pg_stat_statements.mdx

```diff
@@ -44,21 +42,19 @@ track_activity_query_size = 2048
 
 # Track statements generated by stored procedures as well
 pg_stat_statements.track = all
-```
 
-If you're running PostgreSQL 9.2 you may need to increase your OS shared memory
-limits when using pg\_stat\_statements.<br />
-See the Postgres [shared memory documentation](https://www.postgresql.org/docs/9.2/static/kernel-resources.html#SYSVIPC)
-for details on how to do this.
+# Increase the maximum number of statements tracked
+pg_stat_statements.max = 10000
```

Comment:
I've added the example of `pg_stat_statements.max = 10000`. This is the number used in the official doc too, so I think it's okay to say it like this way.